src/stream/src/executor_v2/mview/state.rs

```diff
@@ -66,10 +66,8 @@ impl<S: StateStore> ManagedMViewState<S> {
         assert_eq!(self.order_types.len(), pk.size());
         assert_eq!(self.column_ids.len(), value.size());
 
-        let l = pk.0.len() == 3
-            && pk
-                .0
-                .iter()
+        let l =
+            pk.0.iter()
```

Comment:
Add some comments here, 👀?

Reply:
This is for debugging. 😢 I'll remove it when it's ready.